dev/contr.md

```diff
@@ -0,0 +1,17 @@
+# Contributing
+
+All contributions must be made via a Pull Request to the main GitHub-Repository.
+
+For a pull request to get merged, the following conditions must be fulfilled:
+
+- all applicable pipelines must succeed
+- at least two approvals must are needed for the patch to be merged after 24 hours, or at least four approvals  to merge the patch immediately
+- the PR must neither be controversial or blocked (indicated by tags)
+
+**Fundamental changes, like changes in the database scheme or other disputed areas, require at least four reviews. Those changes should be discussed in a development meeting. A "Fundamental"-Tag should indicate those changes.**
+
+The Documentation may be also expanded using pull requests (PRs). Pull requests need need to follow a few guidelines:
+
+- A PR may be merged immediately with a minimum of only two approvals.
+- Multiple commits must be squashed before or during merging, exceptions confirm the rule.
```

Comment:
```suggestion
- Multiple commits should be squashed before or during merging, except where it would impact comprehensibility.
```

Reply:
Applied suggestions, as it only affects details. Thanks!